src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Properties/InterceptedProjectProperties/ApplicationPropertyPage/ApplicationFrameworkValueProvider.cs

```diff
@@ -254,7 +254,7 @@ private async Task<bool> IsWPFApplicationAsync(IProjectProperties defaultPropert
             return null;
         }
 
-        private async Task<string> GetPropertyValueAsync(string propertyName)
+        private async Task<string> GetPropertyValueAsync(string propertyName, IProjectProperties defaultProperties)
```

Comment:
Is anything using the new parameter here?

Reply:
Woop, not anymore, let me take it out